tests/xcodeproj/tests.sh

```diff
@@ -2,4 +2,5 @@ set -eux
 
 cd $(dirname $0)
 
-xcodebuild -project Single-Application-Project.xcodeproj -quiet -scheme Single-Application-UnitTests test
\ No newline at end of file
+xcodebuild -project Single-Application-Project.xcodeproj -quiet -scheme Single-Application-UnitTests test
+xcodebuild -project Single-Application-Project.xcodeproj -quiet -scheme Single-Application-RunnableTestSuite test
```

Comment:
What's the diff between runnable and unrunnable?

Reply:
Ah, please disregard that for now, I removed the Unrunnable-Test-Suite in my latest changes.

The runnable test suite specifies a non-empty list for the attribute called `runners`. A runner is a combination of device-type and os-version (i.e. `iPhone6-9.3` or `iPad-Air-11.4`).

The Unrunnable test suite was testing what happens when the user specifies a target whose runners attribute is an empty list. After I observed that `bazel build` would fail on the Unrunnable-Test-Suite bazel target, I didn't see a great way of testing that case, so I removed that test.